public/index.html

```diff
@@ -9,8 +9,8 @@
   <script>
     document.querySelector('body').style.overflow = 'hidden'
   </script>
-  <script>mainJs</script>
   <div id="root"></div>
+  <script>mainJs</script>
```

Comment:
oh I did not know it would be as simple as this since I checked this file as well.. why do we need to move this line?

Reply:
@QilongTang the mainJS text is replaced on runtime by .NET when loading in WebView2, so in runtime React (js) is trying to find the "root" element inside the html but is not found due that was declared after the <script> tag so just moving the root element one level up will fix the problem.
This is the React code trying to find the root:
![image](https://user-images.githubusercontent.com/61755417/196984403-69524d3d-bc44-4731-a680-362b7ea3c25c.png)